src/web/command_control/client/components/CommandControl.tsx

```diff
@@ -1231,16 +1231,16 @@ export default class CommandControl extends React.Component {
 	// 
 	async deleteAllRunsInMission(mission: MissionInterface, needConfirmation: boolean, rallyPointRun?: boolean) {
```

Comment:
Since we are updating this function can we update the comment to use the correct /**/